cheri-architecture-tr.tex

```diff
@@ -0,0 +1,5 @@
+%%%% Enable University of Cambridge tech-report formatting
+%%%% Use \ifdefined\trformat in other files to check the mode
+\def\trformat{}
+
+\input{cheri-architecture}
```

Comment:
For my thesis I used latexmk's pretex flag: `-pretex='\\newcommand*{\\buildtechreport}{yes}'`, but I guess adding a separate file might be simpler.

Reply:
Ah, that's one of the options I didn't find on stack exchange.  I do think the separate file makes the Makefile change simpler perhaps?  However, it is true that we could use 'make USE_TR=yes' or some such and add this to the flags passed to latexmk if that is defined?  (The other approaches were not quite as straightforward to use with a make variable like USE_TR which is why I ended up going with a separate file.)  I don't have a strong preference either way.